front/src/applications/editor/tools/routeEdition/components/EditRouteMetadata.tsx

```diff
@@ -8,7 +8,7 @@ import { cloneDeep, first, isEqual, last } from 'lodash';
 import { FaFlagCheckered } from 'react-icons/fa';
 import { BsArrowBarRight } from 'react-icons/bs';
 import { MdDelete, MdSave } from 'react-icons/md';
-import { FiSearch } from 'react-icons/fi';
+import { FiPlusCircle } from 'react-icons/fi';
```

Comment:
Could you use `GoPlusCircle` instead ? we actually try to only use Github Octicons, it'll be the only style we'll allow (new icons will be designed by our UX/UI with the same guidelines). Thx!